.github/workflows/gradle-precommit.yml

```diff
@@ -0,0 +1,23 @@
+name: Gradle Precommit
+
+on: 
+  pull_request:
+    branches:
+    - '*'
+
+jobs:
+  test:
+    name: gradle precommit w/ Java 11
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
```

Comment:
gradlew should have this permission already when you do a git clone? Why is it explicit?